README.md

```diff
@@ -30,125 +29,87 @@ We are working under an agile framework, and plan to use this repo to publish, r
 
 1. Run:
-	```
-	$ pip install virtualenv virtualenvwrapper
+	```sh
+	pip install virtualenv virtualenvwrapper
 	```
 
 ### Autoenv module
 
 If you already have [Autoenv](https://github.com/kennethreitz/autoenv) installed, [skip ahead to Sheer](#sheer-elasticsearch).
 
 1. Run:
+	```sh
+	pip install autoenv
 	```
-	$ pip install autoenv
-	```
-
-
-### Sheer & Elasticsearch
-
-[Sheer](https://github.com/cfpb/sheer) is "A Jekyll-inspired, elasticsearch-powered, CMS-less publishing tool." It requires [Elasticsearch](http://www.elasticsearch.org/).
-
-1. Install [Elasticsearch](http://www.elasticsearch.org/) however you'd like. (We use [homebrew](http://brew.sh/).):
-	```
-	$ brew install elasticsearch
-	```
-
-2. Clone the sheer GitHub project to wherever you keep your projects (not inside Owning a Home):
-	```
-	$ git clone https://github.com/cfpb/sheer.git
-	```
-
-3. Create a virtualenv for sheer, which you'll name `OAH`:
-	```
-	$ mkvirtualenv OAH
-	```
-
-	The new virtualenv will activate right away. To activate it later on (say, in a new terminal session) use the command `workon OAH`. You'll know you have a virtual environment activated if you see the name of it in parentheses before your terminal prompt. Ex:
-	```
-	(OAH)username$
-	```
-
-	If the virtualenv did not activate right away, run this command from the directory above where the OAH directory lives:
-	```
-	$ source OAH/bin/activate
-	```
-
-	To deactivate, use the command `deactivate`.
-
-4. Install sheer into the virtualenv with the `-e` flag (which allows you to make changes to sheer itself). The path to sheer is the root directory of the GitHub repository you checked out (cloned) earlier, which likely will be `./sheer`:
-	```
-	$ pip install -e ~/path/to/sheer
-	```
-
-5. Install sheer's python requirements:
-	```
-	$ pip install -r ~/path/to/sheer/requirements.txt
-	```
-
-6. You should now be able to run the sheer command:
-	```
-	$ sheer
-	usage: sheer [-h] [--debug] {inspect,index,serve} …
-	sheer: error: too few arguments
-	```
-
-7. Populate Elasticsearch with content:
-  ```
-  $ [WORDPRESS=http://wordpress.uri] sheer index [--reindex]
-  ```
-  Make sure to be in the site's root folder and that Elasticsearch is running. The above
-  command will read all OaH content and save it in Elasticseach. [More information](#wordpress-indexing).
-
-If you run into problems or have any questions about Sheer, check out [Sheer on Github](https://github.com/cfpb/sheer) and the [Sheer Issue Tracker](https://github.com/cfpb/sheer/issues).
 
 ### Node, Grunt, Bower, Browserify
 
 1. Install [node.js](http://nodejs.org/), either v0.12 or io.js. If you're using the default version that comes with Mac OSX, you'll need to upgrade - first [install NVM](https://github.com/creationix/nvm).
 1. If you installed with NVM, then you can install and set io.js as your default node version:
-	```
-	$ nvm install iojs
-	$ nvm alias default iojs
+	```sh
+	nvm install iojs
+	nvm alias default iojs
```

Comment:
Is it worth assuming that Node will be covered, since they had to install cfgov-refresh, and removing this (outdated) guidance?